docs/source/setup_lora.rst

```diff
@@ -21,152 +21,151 @@ first be created at TTN (if one does not already exist).
 Create TTN account
 ^^^^^^^^^^^^^^^^^^
 
-At https://account.thethingsnetwork.org/register you have to enter a
-**USERNAME**, the **EMAIL ADDRESS** and a **PASSWORD**. Then right down
-over **Create account** create the account. After that you can log in to
-the console with the new data
-(https://account.thethingsnetwork.org/users/login).
+At https://account.thethingsnetwork.org/register you must enter a
+**USERNAME**, the **EMAIL ADDRESS** and a **PASSWORD**. Then create the account by clicking on
+**Create account**.  After that you can log in to
```

Comment:
Then click on **Create account**.

Guess you don't need to say again what that does.